articles/install-guide/index.md

```diff
@@ -9,7 +9,7 @@ uid: microsoft.quantum.install
 
 # Getting Started with the Microsoft Quantum Development Kit #
 
-A great way to get started with the Quantum Development Kit is via some of the Jupyter notebooks available in our [GitHub Quantum repository](https://github.com/Microsoft/Quantum.git).  In particular:
+A great way to get started with Q# before installing the Quantum Development Kit is via some of the Jupyter notebooks available in our [GitHub Quantum repository](https://github.com/Microsoft/Quantum.git).  In particular:
```

Comment:
The links below this paragraph are to directly to the GitHub pages for each notebook, rather than to a Binder or Azure Notebooks instance that allows users to run them. Do we want to modify the links below to point to Binder instead?